docs/content/en/schemes/oauth2.md

```diff
@@ -15,6 +15,12 @@ category: Schemes
 this.$auth.loginWith('social')
 ```
 
+Additional arguments can be passed through to Google as the `params` key of the second argument:
```

Comment:
Maybe here we can remove the word "Google", as this example refers to oauth2 scheme. What do you think?

> Additional arguments can be passed through as the `params` key of the second argument:

Reply:
Good find. Changed!